spec/support/features/session_helpers.rb

```diff
@@ -1,19 +1,19 @@
 module Features
   module SessionHelpers
     def sign_in_to_admin_area(user: create(:admin_user))
-      visit admin_login_url
+      visit new_admin_user_session_url
 
       fill_in 'admin_user[username]', with: user.username
       fill_in 'admin_user[password]', with: user.password
 
-      click_button 'Log in'
+      click_button 'Sign in'
```

Comment:
This test asserts nothing. If at the end a blank page is rendered, it will pass.

Reply:
This is not a test, but a test helper (and legacy one, since specs are in maintenance mode).